compiler/rustc_ast_pretty/src/pprust/state/item.rs

```diff
@@ -167,8 +168,15 @@ impl<'a> State<'a> {
                     def,
                 );
             }
-            ast::ItemKind::Const(def, ty, body) => {
-                self.print_item_const(item.ident, None, ty, body.as_deref(), &item.vis, *def);
+            ast::ItemKind::Const(box ast::ConstItem { defaultness, ty, expr }) => {
+                self.print_item_const(
```

Comment:
`print_item_const` could now take an `ast::ConstItem` as an argument

Reply:
Unfortunately this doesn't work yet. We'll need to merge the shared parts of const items and static items first.